src/sylveon/_classes/_account.py

```diff
@@ -0,0 +1,31 @@
+from bs4 import BeautifulSoup
+import requests
+
+from sylveon._classes._scrape_helpers import scrape_courses_info
+
+
+class Account:
+
+    def __init__(self, session):
+        self.session = session
+
+    def get_courses(self) -> dict:
+        courses = {"instructor": {}, "student": {}}
+
+        endpoint = "https://www.gradescope.com/account"
+
+        # get main page
+        response = self.session.get(endpoint)
+        soup = BeautifulSoup(response.text, "html.parser")
+
+        # get instructor courses
+        instructor_courses = scrape_courses_info(
+            self.session, soup, "Instructor Courses"
+        )
+        courses["instructor"] = instructor_courses
+
+        # get student courses
+        student_courses = scrape_courses_info(self.session, soup, "Student Courses")
```

Comment:
On further testing, I believe this may be the same case for users that are solely instructors too. Will look into how to differentiate these two cases.

Reply:
Oh good point. I think you could differentiate between if an instructor/student is logged in by checking the button in the bottom right. For instructors it's `Create Course` whereas for student accounts it's `Add Course`.

Then you could check both `Your Courses` and `Student Courses`/`Instructor Courses`

Reply:
Yep! That's how I ended up resolving the issue, changes are pushed